modules/rhel-preparing-playbook-machine.adoc

```diff
@@ -55,15 +55,28 @@ If you use SSH key-based authentication, you must manage the key with an SSH age
 # subscription-manager attach --pool=<pool_id>
 ----
 
-. Enable the repositories required by {product-title} {product-version}:
+. Enable the repositories required by {product-title} {product-version}.
+
+.. For {op-system-base} 7 nodes, you must enable the following repositories:
```

Comment:
If we deprecated RHEL-7 workers in OCP-4.9, shall we remove the RHEL-7 worker scaling-up section?